release/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GO_VERSION=1.14
+ARG GO_VERSION=1.16
```

Comment:
Wouldn't this affect the min go version requirements? So would need to bumb up the required Go version? Or am I mistaken?